mods/cubyz/rotation/log.zig

```diff
@@ -179,7 +179,7 @@ pub fn createBlockModel(_: Block, _: *u16, _: ZonElement) ModelIndex {
 }
 
 pub fn model(block: Block) ModelIndex {
-	return .{.index = blocks.meshes.modelIndexStart(block).index + (block.data & 63)};
+	return @enumFromInt(@intFromEnum(blocks.meshes.modelIndexStart(block)) + (block.data & 63));
```

Comment:
Since this is done in all rotation modes, should we maybe consider adding an `add` function to make this less verbose?